.coveragerc

```diff
@@ -0,0 +1,16 @@
+[run]
+source = seirmo
+omit = 
+    seirmo/tests/*
```

Comment:
Could you also add the version_info.py here, i.e.
```
omit = 
    seirmo/test/*
    seirmo/version_info.py
```

Reply:
I have done those:) Thank you!